src/firefly/js/ui/Menu.jsx

```diff
@@ -403,79 +403,87 @@ function SideBarView({menu,appTitle,closeSideBar,haveResults,selected,dropDown,
-                    </Tooltip>
-                }
-            </Stack>
-
-            <Stack spacing={2}>
-                {/* No-category Group */}
-                <Stack spacing={groupSpacingY}>
-                    {/* Results */}
-                    <ResultsTip>
-                        <ToolbarButton {...{
-                            pressed: !selected && !dropDown.visible,
-                            text: (
-                                <Stack direction='row' spacing={.5} alignItems='center'>
-                                    <InsightsIcon color='success'/>
-                                    <Stack direction='row' spacing={1} alignItems='baseline'>
-                                        <Typography color='success' /*sx={{color:'unset'}}*/>Results</Typography>
-                                        {!haveResults && <Typography level='body-xs'>(No results yet)</Typography>}
-                                    </Stack>
-                                </Stack>
-                            ),
-                            onClick: () => {
-                                dispatchHideDropDown();
-                                closeSideBar();
-                            },
-                            sx:(theme) =>( {
-                                    '& .ff-toolbar-button': {color:theme.vars.palette.success.solidBg },}
-                            )
-                        }} />
-                    </ResultsTip>
-
-                    {/* Upload */}
-                    {uploadItem &&
-                        <SideBarItem {...{key:'UPLOAD', item:uploadItem,selected,
-                            menu,closeSideBar,allowMenuHide, sx:itemLayoutSx}}/>
-                    }
-
-                    {/* Other no-category items, if any */}
-                    {Boolean(noCatItems?.length) &&
-                        menuItems
-                            .filter( ({category}) => !category )
-                            .map( (item) => (<SideBarItem {...{key:item.label, item,selected,menu,closeSideBar, allowMenuHide, sx:itemLayoutSx}}/>) )
+        <Stack mt={1/2}>
+            <Divider orientation='horizontal'/>
+            <Sheet variant='soft'>
+                <Stack direction='row' sx={{py: '.25rem', px:'.75rem',...itemLayoutSx}}>
+                    <Typography level='h4' sx={{minWidth: itemMinWidth}}>Tab Selection</Typography>
+                    {showReset &&
+                        <Tooltip title={'Reset visibility of all the tabs to default'}>
+                            <Chip
+                                sx={{mr:1/2}}
+                                onClick={() => {
+                                    const menuItems= menu.menuItems.map( (mi) => ({...mi, visible: undefined}) );
+                                    dispatchHideDropDown();
+                                    updateMenu(appTitle, {...menu, menuItems});
+                                }}>
+                                Reset All
```

Comment:
Because of the grey background, this chip is almost indistinguishable  - maybe make it outlined?